docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "2"
+services:
+  app:
+    build: .
+    image: keboola/ex-http
+    tty: true
+    stdin_open: true
+    environment:
+        - KBC_DATADIR=/code/data/
+    volumes:
+        - ./:/code
```

Comment:
tohle nebude fungovat na travisu, protoze se /code/ uvnitr image prekryje adresarem z hosta, kde neni vendor, takze je potreba mit tam 2 servicy - zhruba tahle https://github.com/keboola/generic-extractor/blob/master/docker-compose.yml

Reply:
Fixunto. Díky.

Reply:
více servisám se v docker-compose snažím vyhnout, vypadá to spíš jako antipattern.
používám teď jednu servisu `dev`/`php` (https://github.com/keboola/aws-s3-extractor/blob/master/docker-compose.yml#L3) a testy pouštím jako `docker-compose run --rm dev ./vendor/bin/phpunit`. na tom linku jsou sice dvě servisy, ale ta druhá používá jinej base image a slouží pro generování code coverage, tam je to imho oprávnění.

Reply:
no jo, jenze pak ti jedou testy oproti kodu, kterej na hostovi a ne uvnitr image - kterej by pochopitelne mel byt uplne stejnej, kdyz je ten image z toho zrovna zbuildovanej, ale @ujovlado by z toho nadsenej nebyl :)

Reply:
@ondrejhlavacek tohle se mi teda nelibi https://github.com/keboola/aws-s3-extractor/blob/master/docker-compose.yml#L3
Takhle https://github.com/keboola/docker-demo-app/blob/master/docker-compose.yml by to melo byt vsude. Kde to tak neni opravuju to na to.
viz. https://github.com/keboola/db-extractor-snowflake/pull/39/files

Reply:
Sorry, četl jsem @Halama komentář v mailu a nevšiml si, že @odinuv veskrze shrnul můj hlavní problém s tím, který jsem popsal trochu obsáhleji na slacku. A ještě navíc koukám, že ten v té docker-demo-app je to (imho) špatně a v tom db-extractor-snowflake je to tak jak si myslím, že je to správně.

Reply:
Jo, tak jsme si to vyjasnili na slacku. Výsledek je https://github.com/keboola/docker-demo-app/issues/5 a správně je to podle snowflaku.